packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -36,6 +36,23 @@ const nodeModulesButNotKbnPackages = (path: string) => {
   return !path.includes(`node_modules${Path.sep}@kbn${Path.sep}`);
 };
 
+const mediaAwareStyleLoaders = [
+  {
+    resourceQuery: /print/,
+    loader: 'style-loader',
+    options: {
+      attributes: { 'data-print-media-style': 'true' },
+    },
+  },
+  {
+    resourceQuery: undefined,
+    loader: 'style-loader',
+    options: {
+      attributes: { media: 'screen, projection' },
+    },
+  },
+];
+
```

Comment:
The version of `style-loader` we are using has a known bug for setting the `media` attribute on style tags https://github.com/webpack-contrib/style-loader/issues/458

Reply:
This is a work-around that instead uses the `MutationObserver` hacked into core for updating media attributes based on the presence of the `data-print-media-style`. The result of this is the print styles "flash" once as they are applied by the browser until the `MutationObserver` kicks in. Very hacky, but work-around for rather not updating style-loader + webpack to latest version.